django/core/mail/utils.py

```diff
@@ -18,3 +21,24 @@ def get_fqdn(self):
 
 
 DNS_NAME = CachedDnsName()
+
+
+def check_mail_setting(value):
+    """
+    Check each item in the value to be a tuple of (Full name, email address)
+    """
+    if not value:
+        return False
```

Comment:
If `ADMINS`/`MANAGERS` is empty (`None`, `()`, `[]`) then `mail_admins()`/`mail_managers()` returns without sending any emails so this check is unnecessary.